sc-memory/sc-memory/scs/scs.g4

```diff
@@ -78,10 +83,10 @@ connector returns [std::string text]
     | '-|>' | '<|-' | '-/>' | '</-'
     | '~>' | '<~' | '~|>' | '<|~'
     | '~/>' | '</~' | '_<>' | '_>' | '_<'
-    | '_..>' | '_<..' | '_->' | '_<-'
-    | '_<=>' | '_=>' | '_<=' | '_-|>' | '_<|-'
-    | '_-/>' | '_</-' | '_~>' | '_<~'
-    | '_~|>' | '_<|~' | '_~/>' | '_</~' )
+    | '_..>' | '_<..' | '<.._' | '_->' | '_<-'| '<-_'
+    | '_<=>' | '_=>' | '_<=' |'<=_' | '_-|>' | '_<|-' | '<|-_'
```

Comment:
`<=_ node` is equal to `<= _node` and `<=_node`

Reply:
All `x_` arcs have such behaviour

Reply:
Add tests, it works correctly